queue_1.py

```diff
@@ -0,0 +1,54 @@
+from list import Node, List, ListIterator
+
+
+class Queue(List):
```

Comment:
I suggest not using inheritance. Better make the list a field of the class, and use the public interface of List class only